applications/tari_dan_app_grpc/proto/transaction.proto

```diff
@@ -67,10 +67,10 @@ message MerkleProof {
 message Transaction {
   repeated ThaumInput inputs = 1;
   repeated ThaumOutput outputs = 2;
-  repeated Instruction instructions = 3;
-  bytes balance_proof = 4;
-  tari.dan.common.Signature signature = 5;
-  uint64 fee = 6;
+  repeated Instruction fee_instructions = 3;
+  repeated Instruction instructions = 4;
+  bytes balance_proof = 5;
```

Comment:
I'm assuming this is a remnant of a previous attempt at fees, and can probably be removed